--- create.go
+++ create.go.new
@@ -42,7 +42,7 @@
 			Name:  "no-pivot",
 			Usage: "do not use pivot root to jail process inside rootfs.  This should be used whenever the rootfs is on top of a ramdisk",
 		},
-		cli.BoolFlag{
+		cli.BoolTFlag{
 			Name:  "no-new-keyring",
 			Usage: "do not create a new session keyring for the container.  This will cause the container to inherit the calling processes session key",
 		},
--- run.go
+++ run.go.new
@@ -53,7 +53,7 @@
 			Name:  "no-pivot",
 			Usage: "do not use pivot root to jail process inside rootfs.  This should be used whenever the rootfs is on top of a ramdisk",
 		},
-		cli.BoolFlag{
+		cli.BoolTFlag{
 			Name:  "no-new-keyring",
 			Usage: "do not create a new session keyring for the container.  This will cause the container to inherit the calling processes session key",
 		},
